sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1011,6 +1016,146 @@ If a node containing a dynamic parameter is updated through the whole graph
 update API, then any previous updates to the dynamic parameter will be reflected
 in the new graph.
 
+==== Graph-Owned Memory Allocations [[graph-memory-allocations]]
+
+It can be desirable for a graph to own and manage memory allocations for memory
+associated with commands in the graph. This is made possible by using the
+`async_malloc` and `async_free` functions from the
+link:../experimental/sycl_ext_oneapi_async_memory_alloc.asciidoc[sycl_ext_oneapi_async_memory_alloc
+] extension. These commands can be added to a graph either via queue recording
```

Comment:
```suggestion
] extension. These commands can be added to a graph either via queue recording the async malloc and free enqueue functions
```

Reply:
Is this referring to the free functions that don't return events specifically? I'm not really sure of the need for this clarification here, given we have previously just mentioned the functions by name and the API usage section immediately below this gives examples of both the handler versions and the enqueue functions for both APIs.